cmd/ftl/cmd_call.go

```diff
@@ -58,3 +75,88 @@ func (c *callCmd) Run(ctx context.Context, client ftlv1connect.VerbServiceClient
 	}
 	return nil
 }
+
+// findSuggestions looks up the schema and finds verbs that are similar to the one that was not found
+// it uses the levenshtein distance to determine similarity - if the distance is less than 40% of the length of the verb,
+// it returns an error if no closely matching suggestions are found
+func (c *callCmd) findSuggestions(ctx context.Context, client ftlv1connect.ControllerServiceClient) ([]string, error) {
+	logger := log.FromContext(ctx)
+
+	// lookup the verbs
+	res, err := client.GetSchema(ctx, connect.NewRequest(&ftlv1.GetSchemaRequest{}))
+	if err != nil {
+		return nil, err
+	}
+
+	modules := res.Msg.GetSchema().GetModules()
```

Comment:
Typically we'll convert the `schemapb.Schema` types immediately into `backend/schema.Schema` as this contains useful `String()` methods, as well as helper methods.